app/src/main/java/at/tugraz/ist/swe/ShapeTool.java

```diff
@@ -0,0 +1,113 @@
+package at.tugraz.ist.swe;
+
+import android.graphics.Canvas;
+import android.graphics.Paint;
+import android.graphics.Point;
```

Comment:
Unused Import statement:

`import android.graphics.Point;`

Pls remove it.

Reply:
done